server/container/guice/cassandra/src/main/java/org/apache/james/modules/mailbox/CassandraMailboxModule.java

```diff
@@ -229,6 +229,7 @@ protected void configure() {
             .addBinding().to(MailboxAnnotationListener.class);
         Multibinder.newSetBinder(binder(), EventListener.ReactiveGroupEventListener.class)
             .addBinding().to(DeleteMessageListener.class);
+        Multibinder.newSetBinder(binder(), DeleteMessageListener.DeletionCallback.class);
```

Comment:
Q: what is implementation this binded to? Maybe this is duplicated with the binding declared in `CassandraDeletedMessageVaultModule` ?

Reply:
None. It just creates an empty set for injections. Without this injections fails without the deletedMessageVault module.